### PR TITLE
ANAN DAY 2019 Update

### PR DESCRIPTION
**Pull request by keenthekeen**
Changes:
- Change year in title (2562)
- Update theme color
- Add new poster and instructions on home page
- Add footer
- Use Font Awesome instead of image file

Annotation 2019-04-25 173331

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.scss"
+              "src/styles.scss",
+              "node_modules/@fortawesome/fontawesome-free/css/all.css"
```
Comment (sarunint) [inline angular.json]: Not good for bundle size.

File: src/assets/i18n/th.json [hunk 2]
```diff
@@ -1,11 +1,15 @@
 {
   "common": {
-    "title": "ลงทะเบียนผู้บริจาค โครงการวันอานันทมหิดล คณะแพทยศาสตร์ จุฬาลงกรณ์มหาวิทยาลัย ประจำปี พ.ศ.๒๕๖๑",
+    "title": "ลงทะเบียนผู้บริจาค โครงการวันอานันทมหิดล คณะแพทยศาสตร์ จุฬาลงกรณ์มหาวิทยาลัย ประจำปีพ.ศ. ๒๕๖๒",
```
Comment (sarunint) [inline src/assets/i18n/th.json]: Add space before พ.ศ.?

File: src/assets/i18n/th.json [hunk 3]
```diff
@@ -63,7 +67,7 @@
   },
   "register": {
     "title": "ลงทะเบียนผู้บริจาค",
-    "subtitle": "โครงการวันอานันทมหิดล คณะแพทยศาสตร์ จุฬาลงกรณ์มหาวิทยาลัย ประจำปี พ.ศ.๒๕๖๑",
+    "subtitle": "โครงการวันอานันทมหิดล คณะแพทยศาสตร์ จุฬาลงกรณ์มหาวิทยาลัย ประจำปีพ.ศ. ๒๕๖๒",
```
Comment (sarunint) [inline src/assets/i18n/th.json]: Same

**Comment by keenthekeen** [comment]
> Also, I have an English translation in there. Please find a way to integrate please?

Where is that?

**Comment by sarunint** [comment]
> > Also, I have an English translation in there. Please find a way to integrate please?
> 
> Where is that?

Whoops, sorry.

File: angular.json (hunk 1 above)
Comment (keenthekeen) [reply angular.json]: Import Font Awesome from CDNJS instead, for now.
Considering import only the icons we use in the future.

**Pull request merged**